bundle/README.md

```diff
@@ -79,11 +79,76 @@ By deploying the `reviews` virtual service, you can specify a different behavior
 
 For more information, see [Bookinfo Application](https://istio.io/latest/docs/examples/bookinfo/) in the upstream Istio documentation.
 
-## Gateway Configuration
+After following the instructions to [Deploying the application](https://istio.io/latest/docs/examples/bookinfo/#start-the-application-services), **you will need to create and configure a gateway** for the `bookinfo` application to be accessible outside the cluster.
+
+## Creating and Configuring Gateways
 
 The Sail Operator does not deploy Ingress or Egress Gateways. Gateways are not part of the control plane. As a security best-practice, Ingress and Egress Gateways should be deployed in a different namespace than the namespace that contains the control plane.
 
-You can deploy gateways using either the Gateway API or Gateway Injection methods. Both are well documented in the Istio documentation.
+You can deploy gateways using either the Gateway API or Gateway Injection methods. 
+
+### Option 1: Istio Gateway Injection
+
+Gateway Injection uses the same mechanisms as Istio sidecar injection to create a gateway from a `Deployment` resource that is paired with a `Service` resource that can be made accessible from outside the cluster. For more information, see [Installing Gateways](https://preliminary.istio.io/latest/docs/setup/additional-setup/gateway/#deploying-a-gateway).
+
+To configure gateway injection with the `bookinfo` application, we have provided a [sample gateway configuration](samples/ingress-gateway.yaml?raw=1) that should be applied in the namespace where the application is installed:
+
+1. Create the `istio-ingressgateway` deployment and service:
+
+    ```sh
+    $ oc apply -f ingress/gateway.yaml
```

Comment:
did you mean `ingress-gateway.yaml`?

Perhaps it's better to make the namespace explicit by specifying `-n <app-namespace>`?

Reply:
Good catch on the file name. Agreed on adding the namespace explicitely as well.

Originally I added a full example of installing Bookinfo into the`bookinfo` namespace, but decided it was better to just refer to the upstream docs, which use the Kubernetes default namespace.